brave/brave-core
# Typescript keyrings refactoring

Resolves https://github.com/brave/brave-browser/issues/19171

- Moved keyrings instantiation to file
- Unified keyrings interfaces

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
- Check wallet page and panels correctly shows information from default wallet
- Check wallet can import accounts from hardware wallets Trezor/Ledger
